### PR TITLE
Updates BoxStation and MetaStation Emergency Shuttles and Adds the Rage Cage Emergency Shuttle

### PR DESCRIPTION
**Pull request by ClownMoff**
## About The Pull Request

1 more shuttle
Meta and Box shuttle rework

## Why It's Good For The Game

Meta and Box old and clunky,

0-before

68747470733a2f2f616666656374656461726330372e626c6f622e636f72652e77696e646f77732e6e65742f6d6462322f696d616765732f3136323335363932372f32393038363737353339352f6d2f5f6d6170735f73687574746c65735f656d657267656e63795f

As for the new shuttle 
The rage cage and the casino are fun alternatives to the regular shuttle

The Ultra Raven shuttle is a EXPENSIVE shuttle you call when you are dealing with a major threat and you need a way to 1.- GTFO and 2.- Minimize damage and maximize recovering people. It's the ultimate big thing you can get as crew that doesnt involve admin intervention, but you need 70k to order it. Comes with a insanely equiped medbay and an armory in the brig. I'm down to increase the prize even more, but I think a shuttle like this is an unique addition to the game that would reward shifts that have tons of money

## Testing Photographs and Procedure

<details>
<summary>Screenshots&Videos</summary>

68747470733a2f2f616666656374656461726330372e626c6f622e636f72652e77696e646f77732e6e65742f6d6462322f696d616765732f3136323335363932372f32393036323531373131342f6d2f5f6d6170735f73687574746c65735f656d657267656e63795f

imagen

imagen

</details>

## Changelog
:cl: Varo
add: Added a bew shuttle: THE RAGE CAGE 
tweak: Gives the Meta and Box shuttles some love and a fresh layer of paint
fix: Adds a tiny fan to the Comically Large Escape Pod
/:cl:

### CODE REVIEW

**Comment by TsunamiAnt** [comment]
could you add some oxy canisters to the ultra raven? In case of shuttle bombings the damage itself is usually repaired quickly, but all the air lost cannot be refilled.
Specifically oxy cans, because there's no way you can get full pressure with just a few cans. So pure oxy at lower pressure must do.

**Comment by ClownMoff** [comment]
imagen

Are 2 enogh?

**Review by spockye — CHANGES_REQUESTED** [review]
Starting with the casino.
im sorry but i really dont like it. its golden walls, golden floors, chairs and slot machines spammed everywhere, even infront of the doors, and a box on the side for medical. 
theres no lights, no decalling, no windows, its just a square. personally i think it might be best to remove it from the PR. if not that a complete redo. 

The Ragecage shuttle
im kinda having some of the same problems with this one as i have with the casino. little to no decalling, mostly empty, and its pretty plain. i dont think it really brings much to the table, since we already have the fighting arena on the ruski shuttle. 

Now, on the ultraraven it might just be personal opinions but. 
im not really a huge fan of the idea. its just a larger version of the raven, and the raven is already a amazing and powerful ship. i really dont think this version is necessary. a full armoury, two medbays. and im not a fan of how its laid out. i think it'll just end up being one of those things that cause a lot of issues at the end of rounds. 

im sorry that this review is pretty negative, but i just felt i should say my two cents on the matter. if people disagree with me then thats totally fine!. 

i'll include some issues not stated above under here:

<details>
<summary>requested changes</summary>
<br>

image
firelock in space

image
i dont think airlocks need firelocks

image
i think the turrets are facing the wrong way

image
something weird going on with the walls here

image
why is there a window here?

image

one firelock shutter, one armoury shutter with a blast door under, one with just a shutter and one with just a door? something weird is going on here. 

image
these buttons are floating off the table

image
this room seems pretty empty and pointless, with there being another giant medbay in the middle. 

image
i'd recommend giving your Shutters more unique ID's. hard to find later, and im unsure if that might conflict with other buttons

image
wild pipe

image
airlock isnt access locked

image
this feels VERY empty

image
door stuck

image
potentially moving around in this would be very akward. the floors also hurt my eyes

image
dont put a window here!


</details>

**Comment by ColonelOrion** [comment]
Let's go gambling!

File: _maps/shuttles/emergency/emergency_ultraraven.dmm [hunk 1]
```diff
@@ -0,0 +1,3124 @@
...
+/turf/open/floor/iron/smooth,
+/area/shuttle/escape)
+"aK" = (
+/obj/structure/closet/firecloset,
+/obj/effect/turf_decal/delivery/red,
+/obj/machinery/light{
+	dir = 8
+	},
+/obj/machinery/button/door{
+	id = ""shuttleevashutter"";
```
Comment (MarkusLarsson421) [inline _maps/shuttles/emergency/emergency_ultraraven.dmm]: The cause of the MapDiffBot2 error. Remove the double double quotation marks. Either in the editor or in the text file.

File: _maps/shuttles/emergency/emergency_casino.dmm
Comment (PowerfulBacon) [inline _maps/shuttles/emergency/emergency_casino.dmm:1]: image

Missing window

Comment (PowerfulBacon) [inline _maps/shuttles/emergency/emergency_casino.dmm:1]: image

Not a fan of the design of this area, bring the right side in and make the room smaller

Comment (PowerfulBacon) [inline _maps/shuttles/emergency/emergency_casino.dmm:1]: image

Window with a wall behind it

Comment (PowerfulBacon) [inline _maps/shuttles/emergency/emergency_casino.dmm:1]: The engine layout and shape of this shuttle just don't really work very well, it doesn't look very... shuttle-y

File: code/datums/shuttles.dm [hunk 2]
```diff
@@ -627,6 +634,20 @@
...
+	description = "Makes you feel like Space Vegas. Comes fully equiped."
+	admin_notes = "It's a normal shuttle but full of slot machines and made out of gold."
+	credit_cost = 5000
+
+/datum/map_template/shuttle/emergency/ragecage
+	suffix = "ragecage"
+	name = "THE RAGE CAGE"
+	description = "WE ARE RAGING! WE ARE CAGING! TIME TO UNLEASH ALL THE STRESS GENERATED DURING THE SHIFT! Comes with a Brig, Medbay and Cockpit included."
+	admin_notes = "It's a normal shuttle but it has a rage cage with baseball bats in the middle powered by a PACMAN, plasma included."
+	credit_cost = 7500
```
Comment (PowerfulBacon) [inline code/datums/shuttles.dm]: This needs to be restricted.

File: code/datums/shuttles.dm [hunk 3]
```diff
@@ -627,6 +634,20 @@
...
+	suffix = "casino"
+	name = "The Emergency Casino"
+	description = "Makes you feel like Space Vegas. Comes fully equiped."
+	admin_notes = "It's a normal shuttle but full of slot machines and made out of gold."
+	credit_cost = 5000
+
+/datum/map_template/shuttle/emergency/ragecage
+	suffix = "ragecage"
+	name = "THE RAGE CAGE"
+	description = "WE ARE RAGING! WE ARE CAGING! TIME TO UNLEASH ALL THE STRESS GENERATED DURING THE SHIFT! Comes with a Brig, Medbay and Cockpit included."
```
Comment (PowerfulBacon) [inline code/datums/shuttles.dm]: Tone down the text a little bit. This is a roleplay server and feature still need to make sense in the world.

File: _maps/shuttles/emergency/emergency_ragecage.dmm
Comment (PowerfulBacon) [inline _maps/shuttles/emergency/emergency_ragecage.dmm:1]: Engine pods need a rework, would rather have walls separating them.

File: _maps/shuttles/emergency/emergency_ultraraven.dmm
Comment (PowerfulBacon) [inline _maps/shuttles/emergency/emergency_ultraraven.dmm:1]: Everything about this shuttle just seems so unnecessary, it's far too big and the shape looks like 3 shuttles strapped together.

File: _maps/shuttles/emergency/emergency_meta.dmm
Comment (PowerfulBacon) [inline _maps/shuttles/emergency/emergency_meta.dmm:1]: Big fan of this shuttle, really like the changes and its a big improvement. Seems like there are some issues with the engine block having some missing engines.

File: _maps/shuttles/emergency/emergency_box.dmm
Comment (PowerfulBacon) [inline _maps/shuttles/emergency/emergency_box.dmm:1]: Going to go ahead and say that I don't like this new shuttle design as a default shuttle, default shuttles should be very blank and boring and exist for the purpose of escaping the station and that alone. The size needs to be significantly reduced and the shuttle needs to be more standard.

My immediate suggestions would be:
- Cut the width of the shuttle down from the bar point onwards.
- Move the bridge over into the space where the T shaped walkway is, to support the size reduction of the shuttle.
- Remove the middle seats, and have 2 rows of vertical seats.

**Review by PestoVerde322 — APPROVED** [review]
Seems all good to go, approved.

**Comment by Rukofamicom** [comment]
This PR contains undocumented changes.

A fresh coat of paint should not include new items without documenting them for proper consideration. Making undocumented changes can result in repo bans if it is determined the information was intentionally withheld. 

I'm referring to the expansion of medbay capabilities via adding defib.

**Pull request merged**